content/docs/capabilities/getting-users-identity.md

```diff
@@ -220,6 +220,12 @@ The Pomerium JWT contains at least the following claims:
 | `groups` | The user's group memberships (if supported for the identity provider). |
 | `name` | The user's full name, as specified by the identity provider. |
 
+:::tip Prevent session replay attempts
+
+The `jti` claim (the JWT ID) contains a unique identifier assigned to each Pomerium JWT. You can prevent session replay attempts by implementing a system that monitors `jti` reuse either in real time or by checking session logs.
```

Comment:
This might come across as a little prescriptive.  Wondering if we should change it to "One way you can prevent session replay attempts is by..."

Reply:
Might make sense to distinguish two cases: prevent vs. detect. If you can check the `jti` value in real time, you may be able to use it to _prevent_ a replay attempt. _Or_, if you log the `jti` value, you can at least _detect_ replayed JWTs after the fact.

Reply:
I'll work with that wording. Thanks for the suggestion.